pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -206,6 +238,31 @@ func (db *DB) GetBandwidthAllocationBySignature(signature []byte) ([][]byte, err
 	return agreements, nil
 }
 
+// GetBandwidthAllocations all bandwidth agreements and sorts by satellite
+func (db *DB) GetBandwidthAllocations() (map[string][]*Agreement, error) {
+	defer db.locked()()
+
+	rows, err := db.DB.Query(`SELECT * FROM bandwidth_agreements ORDER BY satellite`)
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		_ = rows.Close()
```

Comment:
What with this error?

Reply:
Egon had written a function that didn't check this error @egonelbre also. I assumed that it's because it's in the defer and we should probably log it instead of not doing anything